INSTALL.md

```diff
@@ -0,0 +1,42 @@
+# Installing YTL Linux
+
+YTL Linux installs:
+ * Ubuntu server with Cinnamon desktop environment. Cinnamon was selected
+   as it might be more familiar with Windows users than some other desktop
+   environments.
+ * A metapackage `ytl-linux-customize` installs a tested Oracle VirtualBox
+   version and some smaller tools. It also contains
+ * Users for the school and MEB. However, the MEB account is currently disabled.
+
+## Instructions
+
+ 1. Download [the installation image](https://linux.abitti.fi/ytl-install.iso).
+ 1. Write the image to an USB stick.
+ 1. The installation is silent and it overwrites the storage. Make sure your disks do not contain any valuable data.
+ 1. Connect the server to a wired network. Your network needs to provide network settings from a DHCP server. If your network requires proxy settings or filters traffic you may encounter problems.
+ 1. Boot the server from the USB stick you created earlier.
+ 1. Select "Install YTL Ubuntu Server" by pressing Enter. After this there is no way back.
+ 1. In an early stage of the installation the server checks the USB stick for defected data. Make sure it reports only 1 corrupted file:\
+ `Check finished: errors found in 1 files!`
```

Comment:
Maybe mention that this glitch might be fixed later (#1)

Reply:
I'll try to keep this installation recipe as short as possible.